extra/catena-message-model-4905-sapflow-node-red.js

```diff
@@ -95,36 +95,33 @@ function Decoder(bytes, port) {
             }
 
             var LiterPerGallon = 3.78541178;
-            var CfPerGallonPropane = 35.97;
-            var CfPerLiterPropane = CfPerGallonPropane / LiterPerGallon;
+            var CfPerGallonSap = 35.97;
```

Comment:
There's a misunderstanding.

Forget about "Cubic feet" -- that's a *gas* concept, and sap is a liquid.

BTU are *totally* irrelevant. We can't convert "sap" to "BTU", it's chalk and cheese.

Reply:
Remove Cf... completely.